### PR TITLE
Merge MOSIP-16473

### PR DESCRIPTION
**Pull request by Ajay1596**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: pre-registration/pre-registration-application-service/src/main/resources/application.properties [hunk 1]
```diff
@@ -30,7 +30,7 @@ audit.service.contextpath: auditmanager
 audit.service.env: http://kernel-auditmanager-service
 audit.url: ${mosip.base.url}/v1/auditmanager/audits
 auth-token-generator.rest.issuerUrl:${mosip.base.url}/keycloak/auth/realms/preregistration
-auth.server.admin.validate.url:https://dev.mosip.net/v1/authmanager/authorize/admin/validateToken
+auth.server.admin.validate.url:https://qa2.mosip.net/v1/authmanager/authorize/admin/validateToken
```
Comment (gsasikumar) [inline pre-registration/pre-registration-application-service/src/main/resources/application.properties]: Why this change? This property file should not be used in the deployment. In case of local run it should be pointing to dev.mosip.net. 

Remove this change

File: pre-registration/pre-registration-application-service/src/main/resources/application.properties [hunk 2]
```diff
@@ -75,7 +75,7 @@ holiday.exceptional.url: ${mosip.base.url}/v1/masterdata/exceptionalholidays
 holiday.url: ${mosip.base.url}/v1/masterdata/getregistrationcenterholidays/
 javax.persistence.jdbc.driver: org.postgresql.Driver
 javax.persistence.jdbc.password: mosip123
-javax.persistence.jdbc.url=jdbc:postgresql://dev.mosip.net:30090/mosip_prereg
+javax.persistence.jdbc.url=jdbc:postgresql://qa2.mosip.net:30090/mosip_prereg
```
Comment (gsasikumar) [inline pre-registration/pre-registration-application-service/src/main/resources/application.properties]: Same as above remove this change

File: pre-registration/pre-registration-application-service/src/main/resources/application.properties [hunk 3]
```diff
@@ -373,8 +373,8 @@ mosip.role.prereg.postloginvalidateotp=REGISTRATION_OFFICER
 mosip.role.prereg.postlogininvalidatetoken=REGISTRATION_OFFICER
 mosip.role.prereg.getconfig=REGISTRATION_OFFICER
 mosip.role.prereg.getrefreshconfig=INDIVIDUAL
-mosip.role.prereg.postnotificationnotify=REGISTRATION_OFFICER
-mosip.role.prereg.postnotification=REGISTRATION_OFFICER
+mosip.role.prereg.postnotificationnotify=REGISTRATION_OFFICER,INDIVIDUAL
```
Comment (gsasikumar) [inline pre-registration/pre-registration-application-service/src/main/resources/application.properties:376]: Is the postnotification API called from UI or within the server?

File: pre-registration/pre-registration-application-service/src/main/resources/application.properties [hunk 4]
```diff
@@ -384,4 +384,5 @@ mosip.role.prereg.getsyncpreregistrationid=REGISTRATION_OFFICER,INDIVIDUAL
 mosip.role.prereg.getsyncpreregistrationidmachineid=REGISTRATION_OFFICER,INDIVIDUAL
 mosip.role.prereg.postsyncconsumedpreregids=REGISTRATION_OFFICER,INDIVIDUAL
 
+spring.cache.type=none
```
Comment (gsasikumar) [inline pre-registration/pre-registration-application-service/src/main/resources/application.properties]: I heard from mayura that this was not tested?

**Review by gsasikumar — CHANGES_REQUESTED** [review]

**Review by gsasikumar — APPROVED** [review]
